esp8266-fastled-webserver/Fields.cpp

```diff
@@ -359,7 +382,7 @@ String getFieldsJson(const Field (&fields)[N]) {
       }
     }
 
-    if (field.type == NumberFieldType) {
+    if (field.type == NumberFieldType || field.type == UtcOffsetIndexFieldType) {
```

Comment:
OK, a new type is used here because this defines the type of UI associated with the type, and you want the timezone to have unique UI.  Makes sense.

At the same time, I would recommend changing `min` and `max` to store `int32_t`, since that's the javascript-native integer type, which also allows storing the offset in minutes directly.